src/index.js

```diff
@@ -239,6 +239,11 @@ class Migrate extends Command {
         filter: glob => globby(glob)
       }]);
 
+      if (!filePaths?.length) {
+        this.log.error('No files found for the passed glob');
+        break;
```

Comment:
We want to move on to the next item in the loop rather than abort the loop entirely.

```suggestion
        continue;
```